subprojects/docs/src/samples/ivy-publish/java-multi-project/publishing_ivy:publish-custom-artifact-snippet.sample.conf

```diff
@@ -0,0 +1,2 @@
+executable: gradle
```

Comment:
Well, this file name contains ":" and broke some windows tests... https://builds.gradle.org/viewLog.html?buildId=13947398&buildTypeId=Gradle_Check_Platform_Java8_Oracle_Windows_integTest

Reply:
https://github.com/gradle/gradle-private/issues/1353